qiskit/ignis/characterization/calibrations/ibmq_utils.py

```diff
@@ -17,11 +17,11 @@
 """
 
 from collections import defaultdict
+
 import numpy as np
-from scipy.optimize import least_squares
-from qiskit.pulse import SamplePulse, pulse_lib, Schedule, FrameChange
+from qiskit.pulse import pulse_lib, Schedule, Play, ShiftPhase
 from qiskit.pulse.schedule import ParameterizedSchedule
-from qiskit.pulse.commands import FrameChangeInstruction
+from scipy.optimize import least_squares
```

Comment:
The scipy import needs to be moved before the qiskit imports to fix linter errors